### DISCUSSION TITLE
Galician Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#583] [Language Leaders]
Starting discussion about Galician language leader(s). @cibernox I will add you :+1:

**Comment by cibernox** [comment]
Thanks!

**Comment by synesthesiam** [comment]
@cibernox Would you be able to translate the new responses files into Galician?

**Reply by cibernox** [reply]
@synesthesiam I can, but I can't make promises until Friday or so.

**Reply by synesthesiam** [reply]
Sure, no rush. The HA beta comes out the week, so we're just gathering up sentences/responses to go into the next version. Worst case, it will be in next month's version :slightly_smiling_face:
